### PR TITLE
check if NotAfter key is present before proceeding to cert renewal

### PR DESCRIPTION
**Pull request by ideahitme**
**DO NOT MERGE YET**

/cc @tuxlife please have a look

### CODE REVIEW

File: sevenseconds/config/acm.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def configure_acm(account: object, region):
                             )
                 elif cert['Status'] != 'ISSUED':
                     continue
-                if datetime.timedelta(weeks=4) > cert['NotAfter'] - datetime.datetime.now(cert['NotAfter'].tzinfo):
+                if 'NotAfter' in cert and datetime.timedelta(weeks=4) > cert['NotAfter'] - datetime.datetime.now(cert['NotAfter'].tzinfo):
```
Comment (tuxlife) [inline sevenseconds/config/acm.py]: Please change the `if` to an `elif` and remove the `'NotAfter' in cert and `

**Review by tuxlife — CHANGES_REQUESTED** [review]

File: sevenseconds/config/acm.py (hunk 1 above)
Comment (ideahitme) [reply sevenseconds/config/acm.py]: thanks, will do

Comment (ideahitme) [reply sevenseconds/config/acm.py]: done, please have a look

**Review by tuxlife — APPROVED** [review]

**Comment by tuxlife** [comment]
:+1:

**Comment by ideahitme** [comment]
👍

**Pull request merged**